cmd/finish/main.go

```diff
@@ -107,6 +111,28 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	kubernetesClient, err := kubernetes.NewInClusterClient(&kubernetes.ClientConfig{
+		Namespace: ctxt.Namespace,
+	})
+	if err != nil {
+		log.Fatalf("couldn't create kubernetes client: %s", err)
+	}
+
+	notificationClient, err := notification.NewClient(notification.ClientConfig{
+		Namespace: ctxt.Namespace,
+	}, kubernetesClient)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = notificationClient.CallWebhook(context.TODO(), notification.PipelineRunResult{
+		PipelineRunURL: pipelineRunURL,
+		OverallStatus:  opts.aggregateTasksStatus,
+		ODSContext:     ctxt,
+	})
+	if err != nil {
+		log.Printf("Calling notification webhook failed: %s", err)
```

Comment:
If I'm not mistaken, this line is also printed when notification isn't enabled. I think the message should be different in this case.

Maybe the ConfigMap should always be installed, but indicate whether it is enabled or not?